main.tf

```diff
@@ -172,7 +173,7 @@ resource "aws_ebs_volume" "default" {
 }
 
 resource "aws_volume_attachment" "default" {
-  count       = var.ebs_volume_count
+  count       = local.volumes_count
```

Comment:
```suggestion
  count       = local.volume_count
```

Reply:
Makes sense, consistency is important :)